prefixdb.go

```diff
@@ -62,8 +62,8 @@ func (pdb *PrefixDB) Set(key []byte, value []byte) error {
 	if value == nil {
 		return errValueNil
 	}
-	pdb.mtx.Lock()
-	defer pdb.mtx.Unlock()
+	// pdb.mtx.Lock()
```

Comment:
If we're going to remove the mutex, we should delete it, not just comment it out.

Reply:
you got it.